### DISCUSSION TITLE
Canais off line

### DISCUSSION THREAD

**Discussion by cportugalr** [LITUATUI/M3UPT#48] [General]
Os canais Sic Notícias e RTP Internacional estão off no kodi.

**Comment by openseabrus** [comment]
Sic Notícias já foi discutido [aqui](https://github.com/LITUATUI/M3UPT/issues/43)

**Comment by cportugalr** [comment]


**Reply by LITUATUI** [reply]
Está a funcionar nos leitores de IPTV recomendados.

**Comment by cportugalr** [comment]
Ok. Vou tentar outra vez no kodi.

Porque neste leitor iptv não funciona
https://play.google.com/store/apps/details?id=com.pecana.iptvextreme

A sábado, 31/08/2024, 14:33, Pedro Lima ***@***.***> escreveu:
